jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseProjectServiceImpl.java

```diff
@@ -140,48 +149,39 @@ public void configureNewCaseProject(WorkspaceProject project) {
                              "Updated with case project configuration");
         logger.debug("Updated deployment model saved");
 
-        addWorkDefinitionsFileToProjectResources(kieModule, project, separator);
-    }
-
-    public void addWorkDefinitionsFileToProjectResources(KieModule kieModule, WorkspaceProject project, String separator) {
-        String resourcesPathStr = Paths.convert(kieModule.getKModuleXMLPath()).getParent().getParent().toUri().toString();
-        Path widFilePath = ioService.get(URI.create(resourcesPathStr + separator + WORK_DEFINITION_FILE));
         logger.debug("Adding WorkDefinition.wid file to resources folder {} of the project {}",
-                     widFilePath,
+                     globalPath,
                      project);
-        addWorkDefinitions(widFilePath);
-    }
+        addWorkDefinitions(kieModule, separator);
 
-    public void configurePackage(@Observes NewPackageEvent pkg) {
-
-        if (isCaseProject(Paths.convert(pkg.getPackage().getModuleRootPath()))) {
-            String resourcesPathStr = Paths.convert(pkg.getPackage().getPackageMainResourcesPath()).toUri().toString();
-            String separator = Paths.convert(pkg.getPackage().getModuleRootPath()).getFileSystem().getSeparator();
-
-            Path resourcesPath = ioService.get(URI.create(resourcesPathStr + separator + WORK_DEFINITION_FILE));
-            addWorkDefinitions(resourcesPath);
-        }
+        logger.debug("Default WorkItemDefinitions added to {}",
+                     globalPath.toString());
     }
 
-    protected void addWorkDefinitions(Path location) {
+    protected void addWorkDefinitions(KieModule kieModule, String separator) {
+        String globalPath = Paths.convert(kieModule.getKModuleXMLPath())
```

Comment:
is there some option for obtaining the root path directly from the kieModule API?

Reply:
Hi @romartin I didn't found better way. If you will take a look above there are some ways to get root folder, but it actually returns root against the repo, so it is just `/` it is ok in some cases but for file creation I have to use the full path with schema location, repository name, space name and so on, so I used this approach. Let me know if you know how to achieve it in a better way. Thank you!

Reply:
ok no worries, I also don't know if there is a better way. That API anyway is a bit old and I think we shouldn't spend much time on figuring out this, let's go ahead with this code and let's see if it works properly anyway.
Thanks!